packages/atomic/cypress/integration/sort-dropdown.cypress.ts

```diff
@@ -70,14 +67,42 @@ describe('Sort Dropdown Component', () => {
     );
   });
 
-  it(`when a criteria is not valid
-    should render an error`, () => {
+  it('Should reflect selected sort on URL', () => {
+    selectOption(criteria[1]);
+    const urlHash = `sortCriteria=${encodeURIComponent(criteria[1])}`;
+    cy.url().should('include', urlHash);
+  });
+});
+
+describe('When sort with invalid criteria option', () => {
+  it('Should render an error when <atomic-sort-criteria> is invalid', () => {
     setUpPage(`
-      <atomic-sort-dropdown>
-        <atomic-sort-criteria</atomic-sort-criteria>
-      </atomic-sort-dropdown>
-    `);
-    cy.wait(500);
+        <atomic-sort-dropdown>
+          <atomic-sort-criteria</atomic-sort-criteria>
+        </atomic-sort-dropdown>
+      `);
+    shouldRenderErrorComponent(sortDropdown);
+  });
+
+  it('Should render an error when criteria is missing', () => {
+    setUpPage(`
+        <atomic-sort-dropdown>
+          <atomic-sort-criteria label="Sort"></atomic-sort-criteria>
+        </atomic-sort-dropdown>
+      `);
     shouldRenderErrorComponent(sortDropdown);
   });
+
+  it('Should render an error when criteria contains invalid character', () => {
+    setUpPage(`
+        <atomic-sort-dropdown>
+          <atomic-sort-criteria criteria="size ascending; date ascending"></atomic-sort-criteria>
+        </atomic-sort-dropdown>
+      `);
+    shouldRenderErrorComponent(sortDropdown);
+  });
+});
+
+describe('When sort has no label', () => {
+  it('Should display default label');
```

Comment:
Should this test be implemented in this PR or a future one?

Reply:
no, in the future one, the real fix is not done yet